controllers/ProjectController.php

```diff
@@ -27,10 +22,11 @@
 use app\models\wsModels\WSConstants;
 
 /**
- * CRUD actions for YiiProjectModel
+ * Implements the CRUD for the projects (ws project model : YiiProjectModel) 
```

Comment:
write "Implements the controller for the Projects and according YiiProjectModel"